config/kafka-config.js

```diff
@@ -13,6 +13,9 @@ process.env.DEFAULT_NOTIFICATIONS_TOPIC;
 const APPLICATION_CONFIG_TOPIC = process.env.APPLICATION_CONFIG_TOPIC || 
 process.env.DEFAULT_APPLICATION_CONFIG_TOPIC;
 
+const IMPROVEMENT_PROJECT_NOTIFICATIONS_TOPIC = 
```

Comment:
@aman-tunerlabs - Please use existing notification topic.

Use flag for inApp - true/false, push -true/false and isForATopic - true/false, in case of true pass topic name